docs/api-docs/management-apis/email-templates/overview.md

```diff
@@ -0,0 +1,133 @@
+# Email Templates Overview
+
+<div class="otp" id="no-index">
+
+### On This Page
+
+- [Editing](#editing)
+- [API](#api)
+- [Overrides](#overrides)
+- [Resources](#resources)
+
+</div>
+
+This article provides and overview of BigCommerce's transactional email templates and API.
+
+## Editing
+
+To edit transactional email templates in a store's control panel, [log in](https://support.bigcommerce.com/s/article/Logging-Into-Your-Store) and navigate to **Marketing** › [**Transactional Emails**](http://login.bigcommerce.com/deep-links/manage/transactional-emails).
+
+Use [handlebars expressions](https://developer.bigcommerce.com/api-docs/store-management/email-templates/handlebars) to localize email templates and access dynamic data (like a customer's first name in an order email).
+
+```html
+<!-- Combined Order Status Email Template -->
+
+ <p>{{ lang 'hello' name=order.customer_name }}. Below are the products you ordered.</p>
+
+{{#each order.products}}
+<tr>
+    <td style="padding:5px; font-size:12px; border-bottom:solid 1px #CACACA"><strong>{{ name }}</strong></td>
+    <td style="padding:5px; font-size:12px; border-bottom:solid 1px #CACACA" width="100" align="center">{{#if sku }}{{ sku }}{{else}}&nbsp;{{/if}}</td>
+    <td style="padding:5px; font-size:12px; border-bottom:solid 1px #CACACA" width="100" align="center">{{ quantity }}</td>
+</tr>
+{{/each}}
+```
+
+[Learn how to edit, test, and preview transactional email templates in the control panel](https://support.bigcommerce.com/s/article/Customizing-Emails).
+
+## API
+
+To manage transactional email templates programmatically, use the [Email Templates API](https://developer.bigcommerce.com/api-reference/store-management/email-templates). For example, to [get a list of email templates](https://developer.bigcommerce.com/api-reference/store-management/email-templates/email-templates/getemailtemplates), send a `GET` request to `/v3/marketing/email-templates`.

```

Comment:
`/v3/marketing/email-templates` is returning a 404 in both Postman and Request Runner, as opposed to the "The route is not found, check the URL" message I get when the endpoint doesn't exist. Maybe it's just temporary?

Reply:
It is correct. Its either not rolled out to your store or the experiment is turned on yet.